### PR TITLE
Add pipeline for testing minimum requirements

### PR DESCRIPTION
**Pull request by florianGla**

### CODE REVIEW

File: .github/workflows/pipeline_min_requirements.yml [hunk 1]
```diff
@@ -0,0 +1,46 @@
+# This Pipeline tests the minimum requirements by running instructions from the docs
```
Comment (jochil) [inline .github/workflows/pipeline_min_requirements.yml:1]:
```suggestion
# This Pipeline tests the minimum requirements by setting up a container with the required versions
```

File: .github/workflows/pipeline_min_requirements.yml [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+      options: --privileged
+    steps:
+      - name: Install dependencies
+        run: |
+          apt update
+          apt install -y cmake llvm-11 clang-11 subversion curl
+          ln -s /usr/bin/clang-11 /usr/bin/clang
+          ln -s /usr/bin/clang++-11 /usr/bin/clang++
+          ln -s /usr/bin/llvm-symbolizer-11 /usr/bin/llvm-symbolizer
+          cmake --version | grep 3.16
```
Comment (jochil) [inline .github/workflows/pipeline_min_requirements.yml:29]:
```suggestion
          # to make sure the version from the apt repository is as expected
          cmake --version | grep 3.16
```

File: .github/workflows/pipeline_min_requirements.yml [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+        shell: bash
+
+      - name: Download & execute latest installer
+        run: sh -c "$(curl -fsSL https://raw.githubusercontent.com/CodeIntelligenceTesting/cifuzz/main/install.sh)"
+
+      - name: Download Example Projects
+        run: svn export https://github.com/CodeIntelligenceTesting/cifuzz.git/trunk/examples
```
Comment (jochil) [inline .github/workflows/pipeline_min_requirements.yml:36]: can you also copy the workaround comment from the original pipeline?

**Review by jochil — APPROVED** [review]
